lib/checks/aria/required-children.js

```diff
@@ -9,7 +9,9 @@ function owns(node, virtualTree, role, ariaOwned) {
 	if (node === null) {
 		return false;
 	}
-	var implicit = implicitNodes(role),
+	var implicit = implicitNodes(role).map(
```

Comment:
CI is failing saying `expected [TypeError: null is not an object (evaluating 'implicitNodes(role).map')] to equal null`.